gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Wed Oct 17 17:27:51 EDT 2018
+#Fri Mar 01 16:33:54 EST 2019
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.4-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.6-all.zip
```

Comment:
is this normally committed? @ningning621 @AAAstorga

Reply:
I think in this case it's fine since I was prompted to update the gradle to 4.6 when I pulled the repo.